tools/export/uvision4.py

```diff
@@ -28,7 +28,7 @@ class Uvision4(Exporter):
     Exporter class for uvision. This class uses project generator.
     """
     # These 2 are currently for exporters backward compatiblity
-    NAME = 'uVision4'
+    NAME = 'uvision'
```

Comment:
The "tool name" must be passed to progen for the build method of Project. In progen, uv4 is called uvision